packages/docs/docs/medications/medication-codes.md

```diff
@@ -0,0 +1,104 @@
+---
+keywords:
+  - rxnorm
+  - ndc
+  - drugs
+  - drug codes
+---
+
+# Medication Code Systems
+
+## Introduction
+
+Medication codes play a fundamental role in crafting prescriptions, medication orders, and formularies. EHRs use standardized codes to describe medications to remove ambiguity, ensure patient safety, and streamline billing and analytics.
+
+In an ideal world, a universal standard of codes would be agreed upon to represent drugs. However, the reality is that there are multiple code systems, each with its unique characteristics and applications.
+
+This guide is dedicated to discussing two of the most significant, non-proprietary code systems in U.S. healthcare: [RXNorm](https://www.nlm.nih.gov/research/umls/rxnorm/overview.html) and [NDC](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory).
+
+- [RXNorm](https://www.nlm.nih.gov/research/umls/rxnorm/overview.html) is an international standard endorsed by standards-compliant bodies such as the US Core, NIH, and CDC.
+- [NDC](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory) (National Drug Code) is a U.S. specific system administered by the Food and Drug Administration (FDA)
+
+While there are other proprietary drug databases with their proprietary code systems, they will not be explored in this guide.
+
+## RxNorm
+
+RxNorm is the preferred code system of the [US Core Medication Profile](https://hl7.org/fhir/us/core/stu3.1.1/StructureDefinition-us-core-medication.html) and the National Institutes of Health (NIH), as well as most FHIR profiles.
+
+RXNorm is a hierarchical code system, that allows for different levels of specificity, depending on the application. The RXNorm vocabulary establishes standard names and identifiers for combinations of ingredients, strengths, and dose forms. This is the information doctors typically include when they write a prescription because they often can’t know the specific product that will be used to fill it.
```

Comment:
describing how these handle brand names or colloquial names might be worth a mention here.  I notice you use "Prozac" as an example.  

Also note how synonyms are handled in this code system.